### ISSUE TITLE
correction

### ISSUE THREAD

**Issue by whoafridi** [NARAYANAREDDY37/Question-Answering-with-a-Fine-Tuned-BERT#1]
Good. 
I found that u r not fine-tuning. just used a pretrained `bert-large-uncased-whole-word-masking-finetuned-squad` model for your covid qa . . 

How to fine-tune with squad like dataset???
